From dea6707510769d0fba936c1be868b123df9bb159 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 28 Feb 2017 16:11:52 +0100
Subject: [PATCH] ip, bridge: document -timestamp option

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1417289
Upstream Status: iproute2.git commit 25bc3d3d4aac6

commit 25bc3d3d4aac6d0ab20a01d922b8e8ad6eb2232e
Author: Satish Ashok <sashok@cumulusnetworks.com>
Date:   Mon Oct 12 09:28:55 2015 -0700

    ip, bridge: document -timestamp option

    This patch documents bridge and ip -timestamp option

    Signed-off-by: Satish Ashok <sashok@cumulusnetworks.com>
---
 man/man8/bridge.8 | 3 +++
 man/man8/ip.8     | 4 ++++
 2 files changed, 7 insertions(+)

diff --git a/man/man8/bridge.8 b/man/man8/bridge.8
index 2f72710..6cbf72b 100644
--- a/man/man8/bridge.8
+++ b/man/man8/bridge.8
@@ -308,6 +308,9 @@ link setting is configured on specified physical device
 .BI master
 link setting is configured on the software bridge (default)
 
+.TP
+.BR "\-t" , " \-timestamp"
+display current time when using monitor option.
 
 .SS bridge link show - list bridge port configuration.
 
diff --git a/man/man8/ip.8 b/man/man8/ip.8
index b1c0bc0..caaa1c1 100644
--- a/man/man8/ip.8
+++ b/man/man8/ip.8
@@ -195,6 +195,10 @@ Set the netlink socket receive buffer size, defaults to 1MB.
 .BR "\-iec"
 print human readable rates in IEC units (e.g. 1Ki = 1024).
 
+.TP
+.BR "\-t" , " \-timestamp"
+display current time when using monitor option.
+
 .SH IP - COMMAND SYNTAX
 
 .SS
-- 
1.8.3.1

